app/views/layouts/design_system.html.erb

```diff
@@ -21,7 +11,25 @@
            environment: environment,
            browser_title: ("Error: " if yield(:error_summary).present?).to_s + sanitize((yield(:page_title).presence || yield(:title))) do %>
 
-  <!-- This element exists to initialise the JS module that configures custom Analytics behaviour -->
+  <%= javascript_tag nonce: true do -%>
+    ;(function (window) {
+      window.GOVUK = window.GOVUK || {}
+      window.GOVUK.vars = window.GOVUK.vars || {}
+      window.GOVUK.vars.extraDomains = [
+        {
+          name: 'whitehall-admin',
+          domains: [document.location.host],
```

Comment:
This is a nice shortcut of enabling the analytics code on any domain where whitehall runs but is this what you really want to do? Our default setup on GOV.UK uses different GA properties for integration, staging and production, so we can clearly separate out our analytics data.

Reply:
I'd assumed that's what the `preview` variable does. In that case what does the preview variable do?

Reply:
It seems that this is how we're currently set up looking at https://github.com/alphagov/whitehall/blob/main/app/assets/javascripts/admin/analytics.js but I will check with the performance analysis team what they would prefer

Reply:
I guess it depends on what your required GA4 setup is. Ours is that we have the three environments ([production](https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/javascripts/govuk_publishing_components/domain-config.js#L22), [staging](https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/javascripts/govuk_publishing_components/domain-config.js#L35), [integration](https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/javascripts/govuk_publishing_components/domain-config.js#L50)) and each one has a separate entry in the domain config (see links).

All three have the same value for `id`, staging and integration each have their own unique values for `auth` and `preview`, whereas production has only `id` (the other two are optional). If your setup is 'all the data from any environment goes to the same place' then that's fine, although that seems like it would be quite polluted (e.g. for development we have separate `auth` and `preview` values).

Reply:
Ah, so are you saying that if I use the same `name` in all environments then the data will end up in a single GA4 property, even if the `auth` and `preview` values are different?

Reply:
I've spoken to the Performance Analysts and in Publishing we have separate tag IDs for each environment - just waiting to be provided with those.

Reply:
Correction, we have a single tag ID and a different preview and auth value for the integration environment. We do not have an analytics environment set up for staging. I have added the appropriate configuration